release-schema.json

```
@@ -37,6 +46,14 @@
             "null"
           ],
           "minLength": 1
+        },
+        "unofficialTranslation": {
+          "title": "Unofficial translation",
+          "description": "A true/false field to indicate if the language of the document is an unofficial translation.",
```

Per the style guide:

```suggestion
          "description": "Whether the document is an unofficial translation.",
```